Tests/ApolloCodegenInternalTestHelpers/DirectiveHelpers.swift

```diff
@@ -0,0 +1,15 @@
+import Foundation
+
+extension String {
+  // This function will only be needed until @defer is merged into the GraphQL spec and is
+  // considered a first-class directive in graphql-js. Right now it is a valid directive but must
+  // be 'enabled' through explicit declaration in the schema.
+  public mutating func appendDeferDirective() {
```

Comment:
Until the `@defer` directive is enabled by default in graphql-js the schema needs to declare it. This function enables us to use local small schemas within tests. We can remove this helper once `@defer` is available to all schemas by default.